lib/workers/repository/update/pr/index.ts

```diff
@@ -348,31 +344,27 @@ export async function ensurePr(
         });
         incLimitedValue(Limit.PullRequests);
         logger.info({ pr: pr.number, prTitle }, 'PR created');
-      }
-    } catch (err) /* istanbul ignore next */ {
-      logger.debug({ err }, 'Pull request creation error');
-      if (
-        err.body?.message === 'Validation failed' &&
-        err.body.errors?.length &&
-        err.body.errors.some((error: { message?: string }) =>
-          error.message?.startsWith('A pull request already exists')
-        )
-      ) {
-        logger.warn('A pull requests already exists');
-        return { type: 'without-pr', prBlockedBy: 'Error' };
-      }
-      if (err.statusCode === 502) {
-        logger.warn(
-          { branch: branchName },
-          'Deleting branch due to server error'
-        );
-        if (GlobalConfig.get('dryRun')) {
-          logger.info('DRY-RUN: Would delete branch: ' + config.branchName);
```

Comment:
When doing a "hide whitespace" review of this PR, I don't see where this if clause was relocated to, so it looks like it's removed. Please double check and confirm that it's intentional

Reply:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/1239644/166655677-a525451a-636d-4888-a35e-5a7ab9408df3.png">